src/js/_enqueues/wp/updates.js

```diff
@@ -1152,19 +1153,21 @@
 				}
 			);
 
-			// Add a notice to the modal's footer.
-			$message.replaceWith( wp.updates.adminNotice( noticeData ) );
-
-			// Send notice information back to the parent screen.
-			noticeTarget = window.parent === window ? null : window.parent;
-			$.support.postMessage = !! window.postMessage;
-			if ( false !== $.support.postMessage && null !== noticeTarget && -1 === window.parent.location.pathname.indexOf( 'index.php' ) ) {
-				noticeTarget.postMessage(
-					JSON.stringify( noticeData ),
-					window.location.origin
-				);
+			if ( 'undefined' === typeof response.configurationData.url ) {
+				// Add a notice to the modal's footer.
+				$message.replaceWith( wp.updates.adminNotice( noticeData ) );
+
+				// Send notice information back to the parent screen.
+				noticeTarget = window.parent === window ? null : window.parent;
+				$.support.postMessage = !! window.postMessage;
+				if ( false !== $.support.postMessage && null !== noticeTarget && -1 === window.parent.location.pathname.indexOf( 'index.php' ) ) {
+					noticeTarget.postMessage(
+						JSON.stringify( noticeData ),
+						window.location.origin
+					);
+				}
 			}
-		} else {
+		} else if ( 'undefined' === typeof response.configurationData.url ) {
```

Comment:
Not certain I understand that else conditional is same as initial conditional.

Reply:
This conditional is an `else` to the `isInModal` conditional on line 1144